### ISSUE TITLE
The Battle For The Undercity (alliance version): King Varian Wrynn doesn't show up (personal bug?)

### ISSUE THREAD

**Issue by Noreyk** [dalaranwow/dalaran-wow#5511]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When taking the portal to Undercity from the king's room at Stormwind, the King Varian Wrynn is not present among the other NPCs, so I can't talk to him and start the invasion event.
Ps: other players said the quest work, so I think it's a bug happening to my character only.

**Expected behaviour**: Tell us what should happen instead
When taking the portal to undercity, King Varian Wrynn should be there among the other NPCs.

**Steps to reproduce the problem**:

1.  Can't tell if it's possible to reproduce, as it seems to be a problem in my character only.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

How it is for me (King Varian Wrynn not present):
![20180614002119_1](https://user-images.githubusercontent.com/40255392/41389991-06c105ec-6f6a-11e8-829e-a240684503e0.jpg)

How it should be (taken from wowhead):
![untitled](https://user-images.githubusercontent.com/40255392/41390003-15118a40-6f6a-11e8-82a8-cdfb8b18a54d.png)

**Comment by Rushor** [comment]
Yeah you neeed to be in the correct phase, or King Varian Wrynn is currently in the respawnphase. Please wait some time and he will reappear.
